cvat-sdk/cvat_sdk/core/uploading.py

```diff
@@ -28,14 +28,23 @@
 TUS_CHUNK_SIZE = 10 * 2**20
 
 
+def _make_request_headers(api_client: ApiClient) -> dict[str, str]:
```

Comment:
This is the same function as in `downloading.py` - could you factor it out?

Reply:
Ok, refactored